modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -58,12 +64,12 @@ ifeval::["{context}" == "about-openshift-sdn"]
 |Hardware offloading|Supported
 endif::[]
 |===
+////
 [.small]
 --
-
 1. Egress router for OVN-Kubernetes supports only redirect mode.
 
-2. IPv6 is supported only on bare metal, vSphere, {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+2. IPv6 single-stack networking is only supported on bare-metal clusters. You must enable IPv6 access for your image registry to prevent network connectivity issues when you need to pull images from the registry.
 
-3. IPv6 single stack is not supported on {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+3. IPv6 dual-stack networking is not supported on {vmw-full}, {ibm-power-name}, {ibm-z-name}, and {rh-openstack} clusters.
```

Comment:
I don't think this is correct based on @cybertron's comment.

Reply:
Thanks, @qiliRedHat . I'm confirming a few things with Ben about this table. I've also been asked to remove it from 4.17+

Reply:
Yes. Point 2 should be IPv6 dual-stack networking is supported... and Point 3 should be IPv6 single stack networking is supported only on baremetal.